Symfony/CS/Fixer/PSR1/Psr4Fixer.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\PSR1;
```

Comment:
Code that follows PSR4 may not follows PSR1, and when one want to use PSR4 in his project we told him that PSR1 is deprecated, but now ask him to use it ?